providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdProvider.java

```diff
@@ -14,6 +14,7 @@
 import dev.openfeature.sdk.Value;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.List;
```

Comment:
could you please check on unused imports ? Also in other changed classe (ex:- FlagdProviderTest.java )

Reply:
Fixed.